<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>724</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#header {
	background-color: #004d99;
	padding-left: 10px;
	margin-left: 10px;
	margin-right: 10px;
}


#title {
	color: white;
	font-weight: bold;
	font-size: 36px;
}

#btnBack {
	background-color: #4db8ff;
	border-radius: 20px;
	color: white;
	margin-top: 10px;
	font-size: 24px;
	height: 50px;
}

#btnBack:hover {
	background-color: #1AA4FF;
}

#mainFrame {
	padding: 10px 0px;
	border: 2px solid #0073e6;
	margin-top: 10px;
	margin-right: 10px;
	margin-left: 3px;
}

#mainFrame QPushButton {
	color: #ffffff;
	background-color:  #004d99;
	border-radius: 15px;
	font-weight: bold;
	font-size: 24px;
	height: 80px;
	padding: 10px 16px;
	margin-bottom: 10px;
}

#mainFrame #btnFinalize {
	background-color: #00CC00;
}

#mainFrame #btnFinalize:hover {
	background-color: #00FF00;
}

#mainFrame QPushButton:hover {
	background-color: #0067CC;
}


#labelSubcategory {
	font-size: 32px;
	font-weight: bold;
	margin-bottom: 10px;
}

#txtEditQuestions {
	margin-left: 10px;
	margin-right: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="header">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="btnBack">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>10</y>
            <width>131</width>
            <height>61</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
          <property name="icon">
           <iconset resource="../qrc/main_menu_rc.qrc">
            <normaloff>:/images/assets/arrow-left.svg</normaloff>:/images/assets/arrow-left.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="logo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../qrc/main_menu_rc.qrc">:/images/assets/logo-white.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="title">
         <property name="text">
          <string>PROJECT: WattWise</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="mainFrame">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="labelSubcategory">
         <property name="text">
          <string>Subcategory Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFrame" name="formFrame">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="btnRandomize">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Randomize</string>
            </property>
            <property name="icon">
             <iconset resource="../qrc/main_menu_rc.qrc">
              <normaloff>:/images/assets/refresh-ccw.svg</normaloff>:/images/assets/refresh-ccw.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btnFinalize">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Finalize</string>
            </property>
            <property name="icon">
             <iconset resource="../qrc/main_menu_rc.qrc">
              <normaloff>:/images/assets/check-circle.svg</normaloff>:/images/assets/check-circle.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPlainTextEdit" name="txtEditQuestions">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>450</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../qrc/main_menu_rc.qrc"/>
 </resources>
 <connections/>
</ui>
